doc/conf.py

```diff
@@ -6,10 +6,44 @@
 https://www.sphinx-doc.org/en/master/usage/configuration.html
 """
 
+import os
 import shutil
 import subprocess
 
 
+# -- Copy example notebooks ---------------------------------------------------
+print("Copy example notebook files")
+# Remove old notebooks
+PATH_TARGET = "usage"
+for root, _, files in os.walk(PATH_TARGET):
+    for notebook in files:
+        if notebook.endswith(".ipynb"):
+            full_path = os.path.join(root, notebook)
+            print("  removing notebook", full_path)
+            os.remove(full_path)
+# Copy notebooks from example directory
+PATH_SOURCE = "../examples"
+for root, _, files in os.walk(PATH_SOURCE):
+    for notebook in files:
+        if ".ipynb_checkpoints" in root:
+            continue
+        if not notebook.endswith(".ipynb"):
+            continue
+        path_from = os.path.join(root, notebook)
+        path_to = os.path.join(PATH_TARGET, notebook)
+        print("  copy", path_from, "to", path_to)
+        shutil.copyfile(path_from, path_to, follow_symlinks=True)
+
+DATA_FILES = [
+    "intensity-recipe.yaml",
+]
+for data_file in DATA_FILES:
+    path_from = os.path.join(PATH_SOURCE, data_file)
+    path_to = data_file
+    print("  copy", path_from, "to", path_to)
+    shutil.copyfile(path_from, path_to, follow_symlinks=True)
+
+
```

Comment:
The copy was necessary, because nbsphinx cant handle "out of source build" nicely right?

Reply:
Yes, it's a required work-around https://github.com/spatialaudio/nbsphinx/issues/170